crates/biome_cli/src/commands/clean.rs

```diff
@@ -0,0 +1,10 @@
+pub use crate::commands::daemon::biome_log_dir;
+use crate::{CliDiagnostic, CliSession};
+use std::fs::{create_dir, remove_dir_all};
+
+/// Runs the clean command
+pub fn clean(_cli_session: CliSession) -> Result<(), CliDiagnostic> {
+    let logs_dir = biome_log_dir();
+    remove_dir_all(logs_dir.clone()).and_then(|_| create_dir(logs_dir))?;
```

Comment:
Why do we immediately recreate the dir? Wouldn't it be automatically recreated the next time you run `biome` anyway? If so, it feels "cleaner" to just remove and leave it at that...

Reply:
If there's a Daemon running, it won't log anything anymore, and users would need to restart it manually to recreate the folder